sympy/polys/series/ringpython.py

```diff
@@ -197,6 +249,36 @@ def _useries_mul_ground(
     return _useries(series, prec, dom, ring_prec)
 
 
+def _useries_square(s: USeries[Er], dom: Domain[Er], ring_prec: int) -> USeries[Er]:
+    """Compute the square of a power series."""
+    return _useries_mul(s, s, dom, ring_prec)
```

Comment:
Maybe it makes sense for this to use `dup_sqr`. Potentially we would want a `dup_series_sqr` function though.